Makefile

```diff
@@ -1,4 +1,7 @@
-fresh: update build
```

Comment:
Nice to have this. But this check should be in "build:" clause, as once checkedout, we would be doing "build" mainly and then sending for review. So this will catch it early. Also, will it run against all the plugins in the glide.yaml?

Reply:
Yes good point and that is the right place to make sure all format issues caught up front. I thought of adding `fmtcheck` as dependency to `build` target, only reason for not doing that way was developer may run `build` more often and didn't want to add extra time with these checks.

I will take care of this change and if there is any other review comment let me know.

Reply:
Done